### PR TITLE
AT 00.02| New Item > Create Pipeline Project

### PR DESCRIPTION
**Pull request by leokonss**
https://trello.com/c/RTrL4ljK

### CODE REVIEW

File: src/test/java/school/redrover/PipelineProject3Test.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import school.redrover.runner.BaseTest;
+
+public class PipelineProject3Test extends BaseTest {
+
+    @Test
+    public void testCreatePipelineProject() {
+        String nameOfProject = "NewPipelineProject";
```
[the remark this answers is not in the document]
Comment (leokonss) [reply src/test/java/school/redrover/PipelineProject3Test.java]: fixed

File: src/test/java/school/redrover/PipelineProject3Test.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+        getDriver().findElement(By.cssSelector(".task-link-wrapper>a[href$='newJob']")).click();
+        getWait5().until(ExpectedConditions.visibilityOf(getDriver().findElement(By.cssSelector("#items"))));
+        getDriver().findElement(By.cssSelector("#name")).sendKeys(nameOfProject);
+
+        getDriver().findElement(By.xpath("//span[.='Pipeline']")).click();
+        getDriver().findElement(By.id("ok-button")).click();
+        getDriver().findElement(By.cssSelector("[name='Submit']")).click();
```
[the remark this answers is not in the document]
Comment (leokonss) [reply src/test/java/school/redrover/PipelineProject3Test.java]: fixed

File: src/test/java/school/redrover/PipelineProject3Test.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        getDriver().findElement(By.cssSelector("#jenkins-head-icon")).click();
+
+        WebElement projectName = getDriver().findElement(By.cssSelector(".job-status-nobuilt td>a>span"));
```
[the remark this answers is not in the document]
Comment (leokonss) [reply src/test/java/school/redrover/PipelineProject3Test.java]: fixed

**Review by ArinaJur — APPROVED** [review]
Thank you! And good job

**Pull request merged**